MKESocial/app/src/main/res/layout/activity_not_found.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="team2.mkesocial.Activities.NotFoundActivity"
+    android:background="@drawable/flag_background_bw">
+
+    <TextView
+        android:id="@+id/textView4"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Error 404 Page Not Found"
+        android:textAlignment="center"
+        android:textAppearance="@style/TextAppearance.AppCompat.Title"
+        android:textColor="@android:color/holo_red_dark"
+        tools:layout_editor_absoluteX="45dp"
+        tools:layout_editor_absoluteY="168dp" />
+
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="215dp"
+        android:layout_height="106dp"
+        android:text="Sorry, looks like the pae you are looking for doesn't exist anymore"
```

Comment:
pae -> page

Reply:
😃 and pushing up the crash results to do a custom page rn